experiments/conductor/cmd/runner/runner.go

```diff
@@ -341,62 +341,62 @@ func RunRunner(ctx context.Context, opts *RunnerOptions) error {
 		}
 	case cmdPushBranch: // 9
 		processors := []BranchProcessor{
-			{Fn: makeReadyPR, CommitMsg: "make ready-pr"},
-			{Fn: pushBranch, CommitMsg: "push branch"},
+			{Fn: makeReadyPR, CommitMsgTemplate: "make ready-pr"},
+			{Fn: pushBranch, CommitMsgTemplate: "push branch"},
 		}
 		processBranches(ctx, opts, branches.Branches, "Pushing Branch", processors)
 	case cmdCreateScriptYaml: // 10
-		processBranches(ctx, opts, branches.Branches, "Script YAML", []BranchProcessor{{Fn: createScriptYaml, CommitMsg: "Create gcloud script.yaml"}})
+		processBranches(ctx, opts, branches.Branches, "Script YAML", []BranchProcessor{{Fn: createScriptYaml, CommitMsgTemplate: "mockgcp: create test script for {{command}}"}})
 	case cmdCaptureHttpLog: // 11
-		processBranches(ctx, opts, branches.Branches, "HTTP Log", []BranchProcessor{{Fn: captureHttpLog, CommitMsg: "Capture HTTP Log for mocks"}})
+		processBranches(ctx, opts, branches.Branches, "HTTP Log", []BranchProcessor{{Fn: captureHttpLog, CommitMsgTemplate: "mockgcp: golden output for TestScripts/mock{{group}}/testdata/{{resource}}/crud"}})
```

Comment:
Other commands face the same issue. For example, "Add proto generation to makefile", "Add generated types". I’m thinking maybe we just prefix every commit in conductor with {{group}} and {{resource}}?

Reply:
That could work, though conductor will evolve over time and I think the messages should be about what we are doing, not the tools we used to create the code.